$OpenBSD: patch-pri_utils_pri,v 1.4 2012/04/05 21:02:25 jasper Exp $

- Remove bundled boost and use the system-wide version instead.

--- pri/utils.pri.orig	Sat Mar 10 16:09:09 2012
+++ pri/utils.pri	Thu Apr  5 22:36:34 2012
@@ -17,8 +17,6 @@
 # $Date: 2012/04/05 21:02:25 $
 # ********************************************************************/
 
-INCLUDEPATH += src/lib/boost_1_43_0
-
 HEADERS += \
 src/utils/abstractimagebutton.h \
 src/utils/abstractstatesbutton.h \
